Profiles - ZIB 2017/zib-MedicationAdministration.xml

```diff
@@ -151,6 +152,16 @@
         <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-MedicationAdministration-DeviatingAdministration" />
       </type>
     </element>
+    <element id="MedicationAdministration.extension:deviatingAdministration.extension">

```

Comment:
@ArdonToonstra , is dit iets van Forge?

Reply:
Yes! The mysterious ways of Forge over time.

Reply:
(Had een kleine notie in de commit message gezet over de changes by Forge)

Reply:
Ja, had ik gezien. Vroeg me alleen af of dit erbij hoorde.